web/yo/app/scripts/directives/realtimestring.js

```diff
@@ -53,6 +53,15 @@ angular.module('oncokbApp')
                         });
                     }
                     scope.$watch('data[key]', function (n, o) {
+                        // Remove blank space character
+                        if (!_.isUndefined(n) && !_.isUndefined(o) && (n.indexOf('&nbsp') > -1 || o.indexOf('&nbsp') > -1)) {
```

Comment:
I think one simple `_.replace` would work?

Reply:
You want to replace `&nbsp` directly?

Reply:
actually `&nbsp;`

Reply:
And this is maybe for later, but the solution would probably just a temporary solution because there may be more HTML reserved characters. But haven't seen any yet.

Reply:
The issued you showed to me is not caused by copy and paste. I think curators type blank space accidentally and approve the change so you can see `&nbsp;` on public website. In that case, we don't need to worry about HTML tags reserved scenario because curators cannot type html. We only need to check if `&nbsp` reserved in typed field and clean it.

Reply:
If it's not introduced in copy/paste, how are the characters introduced?

Reply:
The issue shows `&nbsp` in mutation name. I tried to copy and paste mutation name with blank space and the getString function work, so I guess curators may press blank space during typing and they didn't notice that.